init_db.py

```
@@ -1,83 +1,66 @@
-"""Database bootstrap and session utilities for the production stack."""
-
-from __future__ import annotations
-
-import asyncio
-import logging
+#!/usr/bin/env python3
```

Ruff (0.13.3)

Shebang is present but file is not executable

(EXE001)